runtime/arctic/src/xcm_config.rs

```diff
@@ -1,35 +1,30 @@
 use super::{
-	AccountId, Balance, Balances, Call, Event, Origin, ParachainInfo, ParachainSystem, PolkadotXcm,
-	Runtime, XcmpQueue,
+	AccountId, Balances, Call, Event, Origin, ParachainInfo, ParachainSystem, PolkadotXcm, Runtime,
+	WeightToFee, XcmpQueue,
 };
+use core::marker::PhantomData;
 use frame_support::{
-	match_type, parameter_types,
+	log, match_types, parameter_types,
 	traits::{Everything, Nothing},
-	weights::{IdentityFee, Weight},
```

Comment:
Consider using WeightToFee here, or how do we configure the fee here as opposed to the transaction payment fee?